app/components/howItWorksModal.tsx

```diff
@@ -0,0 +1,125 @@
+              see more or fewer teams (1-20 teams).
+            </li>
+            <li>
+              <strong>Explore Teams:</strong> Click on any team to learn more
+              about them!
+            </li>
+          </ol>
+        </section>
+
+        <section className={styles.section}>
+          <h3 className={styles.sectionTitle}>How Color Matching Works</h3>
+          <p className={styles.paragraph}>
+            We use the <strong>Delta E algorithm</strong>, a scientific method
+            that measures how the human eye perceives color differences. This
+            means we don&apos;t just compare RGB values - we find teams whose
+            colors
+            <em> look</em> similar to your choices, just like your eyes would
+            see them!
+          </p>
+          <p className={styles.paragraph}>The algorithm considers:</p>
+          <ul className={styles.list}>
+            <li>Hue (the actual color)</li>
+            <li>Saturation (how vivid the color is)</li>
+            <li>Brightness (how light or dark it is)</li>
+          </ul>
+        </section>
+
+        <section className={styles.section}>
+          <h3 className={styles.sectionTitle}>Sports & Leagues Included</h3>
+          <p className={styles.paragraph}>
+            Our database includes teams from major sports leagues around the
+            world:
+          </p>
+          <ul className={styles.list}>
+            <li>⚽ Football teams</li>
+            <li>🏀 NBA (Basketball)</li>
+            <li>🏈 NFL (American Football)</li>
+            <li>⚾ MLB (Baseball)</li>
+            <li>🏒 NHL (Hockey)</li>
+            <li>🏏 Cricket teams</li>
+            <li>🏉 Rugby teams</li>
+            <li>And many more!</li>
+          </ul>
+        </section>
+
+        <section className={styles.section}>
+          <h3 className={styles.sectionTitle}>Pro Tips</h3>
+          <ul className={styles.list}>
+            <li>
+              <strong>Try complementary colors</strong> for interesting matches
+              - teams often use colors that pop together!
+            </li>
+            <li>
+              <strong>Use the action buttons</strong> between the color pickers:
+              <ul style={{ marginTop: "0.5rem", marginBottom: 0 }}>
+                <li>
+                  <strong>Swap colors</strong> (↔️) - Switch your primary and
+                  secondary colors instantly
+                </li>
+                <li>
+                  <strong>Random team colors</strong> (🎯) - Pick colors from a
+                  random existing team
+                </li>
+                <li>
+                  <strong>Totally random colors</strong> (❓) - Generate
+                  completely random color combinations
+                </li>
+                <li>
+                  <strong>Reset colors</strong> (⊗) - Clear your selections and
+                  start fresh
+                </li>
```

Comment:
```suggestion
                <li>
                  <strong>Swap colors</strong> <FaExchangeAlt />: Switch your
                  primary and secondary colors instantly
                </li>
                <li>
                  <strong>Random team colors</strong> <FaBullseye />: Pick
                  colors from a random existing team
                </li>
                <li>
                  <strong>Totally random colors</strong> <FaRegQuestionCircle />
                  : Generate completely random color combinations
                </li>
                <li>
                  <strong>Reset colors</strong> <FaRegTimesCircle />: Clear your
                  selections and start fresh
                </li>
```

You need to import the new icons, and add `vertical-align: middle` to them in the css